OOP_Lesson_6/Point.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OOP_Lesson_6
+{
+    class Point:Figure
+    {
+        /// <summary>
+        /// Рисует точку на экране
+        /// </summary>
+        public void Print()
+        {
+            if(IsVisible == true)
```

Comment:
Индусский код. Значение свойства - это уже bool-значение.
```suggestion
            if(IsVisible)
```